src/masonite/scheduling/Task.py

```diff
@@ -0,0 +1,147 @@
+import pendulum
+
+
+class Task:
+
+    run_every = False
+    run_at = False
+    run_every_hour = False
+    run_every_minute = False
+    twice_daily = False
+    run_weekly = False
+
+    _date = None
+
+    name = ""
+
+    def __init__(self):
+        """
+        Should only be on the child class. Also needs to be resolved by the container.
+        """
+
+        pass
+
+    def every(self, time):
+        self.run_every = time
+        return self
+
+    def every_minute(self):
+        self.run_every = "1 minute"
+        return self
+
+    def every_15_minutes(self):
+        self.run_every = "15 minutes"
+        return self
+
+    def every_30_minutes(self):
+        self.run_every = "15 minutes"
+        return self
+
+    def every_45_minutes(self):
+        self.run_every = "15 minutes"
+        return self
+
+    def hourly(self):
+        self.run_every = "1 hour"
+        return self
+
+    def daily(self):
+        self.run_every = "1 day"
+        return self
+
+    def weekly(self):
+        self.run_every = "1 week"
+        return self
+
+    def monthly(self):
+        self.run_every = "1 month"
+        return self
+
+    def at(self, run_time):
+        self.run_at = run_time
+        return self
+
+    def daily_at(self, run_time):
+        return self.daily().at(run_time)
+
+    def handle(self):
+        """Fires the task"""
+
+        pass
+
+    def should_run(self, date=None):
+        """If the task should run"""
+
+        # set the date
+        self._set_date()
+
+        if self._verify_run():
```

Comment:
Maybe here you could return `self._verify_run()` directly ?

Reply:
I like it